### DISCUSSION TITLE
How to disable HTTP/2 in `fetch`?

### DISCUSSION THREAD

**Discussion by berkant** [denoland/deno#14851] [Q&A]
Is there a way to disable HTTP/2 for `fetch` calls? I have an upstream dependency with a possibly broken HTTP/2 implementation, and it is randomly returning `INTERNAL_ERROR` HTTP/2 error code. I figured out the problem goes away if I force HTTP/1 in Go which has something like `GODEBUG=http2client=0` to disable HTTP/2. Is there an equivalent in Deno?

**Comment by jupegarnica** [comment]
I think it is not posible.  But usually is only available using SSL.   So, if posible, fetching to http://...  would be in HTTP/1.1
